Avatar_/avatar/avatar.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bricks component library</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="avatar.css">
```

Comment:
your avatar css is already importing style CSS 
you can import all your component CSS in a main CSS & import main in your style CSS and then import this style CSS in all your HTML files

Reply:
Will do after designing all components after merging them into dev